e2e_test/batch/functions/session_timezone.slt

```diff
@@ -0,0 +1,59 @@
+statement ok
+set timezone = "us/pacific";
+
+# Cast date to timestamptz
+query T
+select '2022-01-01'::date::timestamp with time zone;
+----
+2022-01-01 08:00:00+00:00
+
+# Cast timestamp to timestamptz
+query T
+select '2022-01-01 00:00:00'::timestamp::timestamp with time zone;
+----
+2022-01-01 08:00:00+00:00
```

Comment:
We should display `2022-01-01 00:00:00-08:00` or `2022-01-01 08:00:00+00:00`? Although they are equal.